src/main/java/pl/project13/maven/git/build/TeamCityBuildServerData.java

```diff
@@ -0,0 +1,60 @@
+package pl.project13.maven.git.build;
+
+import org.jetbrains.annotations.NotNull;
+import pl.project13.maven.git.GitCommitPropertyConstant;
+import pl.project13.maven.git.log.LoggerBridge;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Map;
+import java.util.Properties;
+
+public class TeamCityBuildServerData extends BuildServerDataProvider {
+
+  private final Properties systemProperties = new Properties();
```

Comment:
Calling this variable ``systemProperties`` is slightly confusing since the naming creates the illusion it contains some properties from the system package.
Please rename it to ``teamcityBuildProperties``.